reactor-netty/build.gradle

```diff
@@ -42,4 +42,32 @@ javadoc.dependsOn(aggregateJavadoc)
 //add docs.zip to the publication
 publishing.publications.mavenJava.artifact(docsZip)
 
+// Define a task to create a reactor-netty apidocs zip classifier which includes both javadoc, reference doc, and reference pdf if any.
+// this classifier will be published to artifactory using some special artifactory properties so it will be also
+// published to https://docs.spring.io/projectreactor/netty/docs/<version>/ directory.
+// (see releaser.gradle which adds the specific properties metadata (zip.name / zip.deployed=false)
+task apidocsZip(type: Zip, dependsOn: [asciidoctor, asciidoctorPdf]) {
+	archiveBaseName.set("reactor-netty")
+	archiveClassifier.set("apidocs")
+
+	afterEvaluate() {
+		//we copy the pdf late, when a potential customVersion has been applied to rootProject
+		from(asciidoctorPdf) {
+			into("reference/pdf/")
+			rename("index.pdf", "reactor-netty-reference-guide-${rootProject.version}.pdf")
+		}
+	}
+	from(asciidoctor) {
+		includeEmptyDirs = false
+		exclude "**/index.pdf"
+		into("reference/html/")
+	}
+	from(aggregateJavadoc) {
+		into "api"
+	}
+}
+
+//add apidocs.zip to the publication
+publishing.publications.mavenJava.artifact(apidocsZip)
```

Comment:
Does this means that we will publish this artifact to Maven Central?

Reply:
yes, if this is an obstructive issue, is there a way to publish this new artifact to artifactory only ?
or alternatively, maybe there is another way: use rsync in order to push the javadocs/reference/pdf directly to docs.spring.io (then I'll need to explore that approach ...)

Reply:
mmm ... this is interesting: in spring-security, it seems that the docs are deployed in docs.spring.io by doing some ssh commands in groovy code, see this [deployDocs](https://github.com/spring-projects/spring-security/blob/main/buildSrc/src/main/groovy/io/spring/gradle/convention/DeployDocsPlugin.groovy#L54-L80) groovy task in spring-security. maybe we should do the same.

I need to explore this; putting this PR in draft mode for the moment.

Reply:
I could experiment this spring-security ssh commands, I see how it works and could test it using a local ssh server container.
I will close this current PR (because I don't want to loose it), and will create a new one based on the same approach used by spring-security.